### DISCUSSION TITLE
N1盒子无法luci-app，求大佬们帮忙

### DISCUSSION THREAD

**Discussion by Hsiao-Jan** [messense/aliyundrive-webdav#644] [Q&A] [answered]
报错提示：
Collected errors:
 * pkg_hash_check_unresolved: cannot find dependency luci-lua-runtime for luci-app-aliyundrive-webdav
 * pkg_hash_fetch_best_installation_candidate: Packages for luci-app-aliyundrive-webdav found, but incompatible with the architectures configured
 * opkg_install_cmd: Cannot install package luci-app-aliyundrive-webdav.

已经安装了aliyundrive-webdav，运行aliyundrive-webdav -h有对应输入。
这luci-lua-runtime是什么依赖呀。从哪里下载？

**Comment by GitHubmnxz** [comment, marked as answer]
降级到1.10.1就行了

**Reply by Hsiao-Jan** [reply]
多谢了，昨晚没等到答案。。我直接重装了

**Reply by GitHubmnxz** [reply]
app-meta-aliyundrive-webdav | 1.10.2-1 | - | - | 
luci-app-aliyundrive-webdav | 1.10.1 | - | - | 
luci-i18n-aliyundrive-webdav-zh-cn | 1.10.1-1
我试着这样装的

**Comment by messense** [comment]
1.10.3 应该修复了。
